GitUI/CommandsDialogs/FormClone.cs

```diff
@@ -112,6 +112,15 @@ protected override void OnRuntimeLoad(EventArgs e)
                     {
                         string text = Clipboard.GetText(TextDataFormat.Text) ?? string.Empty;
 
+                        // Remove git clone prefix, if any.
+                        // This could be extended in future with a regex.
+                        string gitCloneText = "git clone";
+                        int gitClonePrefix = text.IndexOf(gitCloneText, StringComparison.InvariantCultureIgnoreCase);
+                        if (gitClonePrefix != -1)
+                        {
+                            text = text.Substring(gitClonePrefix + gitCloneText.Length + 1);
+                        }
+
```

Comment:
Currently, the check will be for any "git clone" match (is the substing check correct?)
Untested suggestion - this should be a regexp

```suggestion
                        if (text.StartsWith(gitCloneText, StringComparison.InvariantCultureIgnoreCase))
                        {
                            text = text.Substring(gitClonePrefix.Length);
                        }

```